oracledb-ballerina/build.gradle

```diff
@@ -0,0 +1,426 @@
+                password = System.getenv("packagePAT")
+            }
+        }
+    }
+}
+
+static def checkExecResult(execResult, failText, standardOutput){
+    if (execResult) {
+        if (execResult.getExitValue() != 0) {
+            throw new GradleException('Non-zero exit value: ' + execResult.getExitValue())
+        }
+        if (standardOutput.toString().contains(failText)){
+            throw new GradleException('"' + failText + '" string in output: ' + standardOutput.toString())
+        }
+    } else {
+        throw new GradleException('Returned a null execResult object')
+    }
+}
+
+task createTestDockerImage(type: Exec) {
+    if (!Os.isFamily(Os.FAMILY_WINDOWS)) {
+        def standardOutput = new ByteArrayOutputStream()
+        commandLine 'sh', '-c', "docker build -f $project.projectDir/tests/resources/Dockerfile -t ballerina-oracledb" +
+                " -q $project.projectDir/tests/resources/"
+        doLast {
+            checkExecResult(execResult, 'Error', standardOutput)
+            sleep(10 * 1000)
+        }
+    }
+}
+
+def waitForContainerHealth() {
+    def containerName = "ballerina-oracledb";
+    if (!Os.isFamily(Os.FAMILY_WINDOWS)) {
+        def HEALTH_STATUS = ""
+        def counter = 0;
+        while (counter < 60 && !HEALTH_STATUS.toString().contains("healthy")) {
+            sleep(10000);
+            def output = new ByteArrayOutputStream();
+            try {
+                exec {
+                    commandLine 'sh', '-c',
+                            "docker inspect --format='{{json .State.Health.Status}}' ${containerName}"
+                    standardOutput = output
+                }
+                HEALTH_STATUS = output.toString().trim();
+                if (HEALTH_STATUS.toString().contains("unhealthy")) {
+                    return 1;
+                }
+                if (counter % 6 == 0) {
+                    println("${counter/6}: Waiting for container to be healthy");
+                    println("   HEALTH_STATUS: $HEALTH_STATUS \n");
+                }
+                counter = counter + 1;
+            } catch (err) {
+                println(err);
+                return 1;
+            }
+        }
+        return 0;
+    }
+}
+
+def initializeDatabase() {
+    if (!Os.isFamily(Os.FAMILY_WINDOWS)) {
+        try {
+            return exec {
+                commandLine 'sh', '-c',
+                        "docker exec ballerina-oracledb  /bin/bash -c 'cd home/oracle && source .bashrc; sqlplus /nolog <<< @create-schema.sql'"
+            }.exitValue
+        } catch (all) {
+            println(all);
+            return 1;
+        }
+    }
+}
+
+task startTestDockerContainer(type: Exec) {
+    if (!Os.isFamily(Os.FAMILY_WINDOWS)) {
+        def standardOutput = new ByteArrayOutputStream()
+//        commandLine 'sh', '-c', "docker run --rm -d --name ballerina-oracledb -e ORACLE_PWD=password -p 1521:1521 -t ballerina-oracledb"
```

Comment:
Any reason for this comment?

Reply:
Fixed with b04466c7e3261891042328ee057f4959563e158f